easysocial/easysocial/newsfeed.php

```diff
@@ -110,6 +111,12 @@ public function getStream()
 			$options	=	array('clusterId' => $group_id, 'clusterType' => $clusterType, 'startlimit' => $startlimit, 'limit' => $limit);
 		}
 
+		if ($page_id)
```

Comment:
@Pratikshakhandagale  - you can use `elseif` instead of writing 3 different `if` conditions